### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.iothubprovisioningservices.models.error_details_py3.ErrorDetails
  - azure.mgmt.iothubprovisioningservices.models.error_details_py3.ErrorDetailsException
  fullName: azure.mgmt.iothubprovisioningservices.models.error_details_py3
  langs:
  - python
  module: azure.mgmt.iothubprovisioningservices.models.error_details_py3
  name: error_details_py3
  type: module
  uid: azure.mgmt.iothubprovisioningservices.models.error_details_py3
references:
- fullName: azure.mgmt.iothubprovisioningservices.models.error_details_py3.ErrorDetails
  isExternal: false
  name: ErrorDetails
  parent: azure.mgmt.iothubprovisioningservices.models.error_details_py3
  uid: azure.mgmt.iothubprovisioningservices.models.error_details_py3.ErrorDetails
- fullName: azure.mgmt.iothubprovisioningservices.models.error_details_py3.ErrorDetailsException
  isExternal: false
  name: ErrorDetailsException
  parent: azure.mgmt.iothubprovisioningservices.models.error_details_py3
  uid: azure.mgmt.iothubprovisioningservices.models.error_details_py3.ErrorDetailsException
